underactuated.html

```diff
@@ -611,10 +622,22 @@ <h1>Honda's ASIMO vs. passive dynamic walkers</h1>
 
     </example>
 
-    <p> The underactuated systems are not feedback linearizable.  Therefore,
-    unlike fully-actuated systems, the control designer has no choice but to
-    reason about the nonlinear dynamics of the plant in the control design.
-    This dramatically complicates feedback controller design. </p>
+    <p> Fully-actuated systems are feedback equivalent to $\ddot\bq = \bu$,
+    whereas <i>underactuated systems are not feedback equivalent to $\ddot\bq =
+    \bu$</i>.  Therefore, unlike fully-actuated systems, the control designer
+    has no choice but to reason about the more complex dynamics of the plant in
+    the control design. When these dynamics are nonlinear, this can dramatically
+    complicate feedback controller design.
+    </p>
+
+    <p>A related concept is <a
+    href="https://en.wikipedia.org/wiki/Feedback_linearization">feedback
+    linearization</a>.  The feedback-cancellation controller in the example
+    above is an example of feedback linearization -- using feedback to convert a
+    nonlinear system into a linear one.  Asking whether or not a system is
```

Comment:
I like this, but maybe I'd try to add a clarification. You say that fb cancellation above is a special case of fb linearization. I'd add a sentence (after "into a linear one.") to give a sense of why then fb linearization is more general than that.
Maybe something like: "Feedback linearization concerns the more general question of determining whether there exists a controllable linear system that is feedback equivalent to our system.  Controllability is a weaker concept than full actuation, which we will explore more in depth later in the course. In this sense, asking whether or not ..."